tools/api-markdown-documenter/src/api-item-transforms/helpers/TableHelpers.ts

```diff
@@ -179,6 +180,14 @@ export function createSummaryTable(
 			);
 		}
 
+		case ApiItemKind.Variable: {
+			return createVariablesTable(
```

Comment:
Is it possible to abstract most of the logic to a generic createTable function? Seems like createPackagesTable also takes in a map function and a config, performs checks to add a row etc

Reply:
There is some overlap between the different implementations, but there are also subtle differences. While it would be possible to unify some of the code, I worry that it would make the logic harder to read / reason about. I've considered doing this a couple of times and have ended up deciding not to. Might be worth reconsidering in the future though.

Reply:
That said, performing this runtime mapping isn't really necessary. It should be sufficient to just cast the array. I'll definitely make that change now.